### ISSUE TITLE
[Bug] Message says "Sending" forever

### ISSUE THREAD

**Issue by zawyer1** [moezbhatti/qksms#1146]
I experienced a strange bug today. I received a message, viewed it in the QK Reply window, and then responded inside the QK Reply window and then immediately turned off the screen with the side-button. After about 20 minutes I go back into the app and see that the message is still marked as "Sending". I was able to type and successfully send a new message which immediately showed the "Delivered" message, but the old message was still there saying "Sending". I couldn't re-send it or anything. The only thing that worked was to mark it and delete it.

Not really sure what caused this.

**Comment by moezbhatti** [comment]
Do you know if the recipient ever received the message that was stuck in the "Sending" state?

**Comment by zawyer1** [comment]
No, the message was not received by the recipient. That's how I noticed it hadn't been sent (I didn't get a response).

I normally go back into the app to see that it says "Delivered", but on this particular occasion I was in a hurry and wanted to respond quickly, so I immediately shut off the screen after pressing the Send button. Not sure if it had something to do with that.

**Comment by zawyer1** [comment]
Another thing I'd like to add in regards to this is the convenience of toast messages for delivery confirmations, since you can see the delivery confirmation regardless of whether you're inside the actual app. In the old QKSMS I would have a habit of always waiting 1-2 seconds for the toast message to show up. Since 3.0 doesn't have toast messages it can cause issues like these in rare situations. 👍

**Comment by moezbhatti** [comment]
@zawyer1 definitely :) There's an open issue for it here: https://github.com/moezbhatti/qksms/issues/959

**Comment by seriosbrad** [comment]
I'm also seeing the message sending status stuck on "Sending..." for a very long time.  I'm not using delivery confirmation, and my recipient **is receiving** my messages when it's showing "Sending...."

![vroq6bt](https://user-images.githubusercontent.com/967695/51567982-da000a80-1e55-11e9-98e3-b5f1638203a2.jpg)

Package: com.moez.QKSMS
Version: 3.6.1
Device: lge LM-Q910
SDK: 28
Upgraded
